src/ui/src/builder/settings/BuilderSettingsProperties.vue

```diff
@@ -286,6 +293,28 @@ const errorsByFields = useFieldsErrors(
 	secretsManager,
 );
 
+const selectedCategoryTab = ref<FieldCategory>(FieldCategory.General);
+
+function categoryToLabel(category: FieldCategory) {
+	switch (category) {
+		case FieldCategory.General:
+			return "Configure";
+		case FieldCategory.Style:
+			return "Style";
+		case FieldCategory.Tools:
+			return "Tools";
+		default:
+			return category;
+	}
+}
+
+const fieldCategoryTabOptions = computed<WdsTabOptions<FieldCategory>[]>(() => {
+	return fieldCategories.value.map((category) => ({
+		label: categoryToLabel(category),
+		value: category,
+	}));
+});
```

Comment:
Maybe using a mapping object makes things clearer. It'll also raise a TS warning if we add a new category. WDYT ?
```suggestion
const LABELS_BY_CATEGORY = Object.freeze<Record<FieldCategory, string>>({
  [FieldCategory.General]: "Configure",
  [FieldCategory.Style]: "Style",
  [FieldCategory.Tools]: "Tools",
});

const fieldCategoryTabOptions = computed<WdsTabOptions<FieldCategory>[]>(() => {
  return fieldCategories.value.map((category) => ({
    label: LABELS_BY_CATEGORY[category] ?? category,
    value: category,
  }));
});
```

Reply:
great suggestion, I'm going to apply it!